pkg/apis/velero/v1/pod_volume_restore_type.go

```diff
@@ -98,6 +110,7 @@ type PodVolumeRestore struct {
 }
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root
```

Comment:
Maybe rbac related marks of kubebuilder can be added here.
For example:
// +kubebuilder:rbac:groups=velero.io,resources=downloadrequests,verbs=get;list;watch;create;update;patch;delete
// +kubebuilder:rbac:groups=velero.io,resources=downloadrequests/status,verbs=get;update;patch

Reply:
I'm following the existing style in other Velero code